app/components/draft_scheduled_post_header/draft_scheduled_post_header.tsx

```diff
@@ -135,25 +170,44 @@ const DraftPostHeader: React.FC<Props> = ({
 
     return (
 
-        <View
-            style={style.container}
-            testID={testID}
-        >
-            <View style={style.infoContainer}>
-                {headerComponent}
-                <Text style={style.displayName}>
-                    {channel.displayName}
-                </Text>
+        <View>
+            {draftType === DRAFT_TYPE_SCHEDULED &&
```

Comment:
0/5 I don't see any tests for the new behavior.

Reply:
Same